pkgs/applications/audio/psst/default.nix

```
@@ -0,0 +1,38 @@
+{ pkgs, lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "psst";
+  version = "0.1.0";
+  buildAndTestSubdir = "psst-gui";
+
+  src = fetchFromGitHub {
+    owner = "jpochyla";
+    repo = pname;
+    rev = "f6eb1fe8d273af43266407d998aacac87df8c04d";
+    sha256 = "0w939w7sdalkms63dzgq6s12hwxngbhbqy7rvnq4wyn10g8hv5y1";
+  };
+
+  cargoSha256 = "1zrdd82rdshhczlyndn0g847jq0f801sj27kggyky4jxzl89p28s";
+
+  meta = with lib; {
+    description = "Fast and multi-platform Spotify client with native GUI";
+    homepage = "https://github.com/jpochyla/psst";
+    license = licenses.mit;
+    maintainers = [ maintainers.vbrandl ];
+  };
+
+  buildInputs = with pkgs; [
+    alsa-lib
+    atk
+    cairo
+    dbus
+    gdk-pixbuf
+    glib
+    gtk3
+    pango
+  ];
+
+  nativeBuildInputs = with pkgs; [
```

```suggestion
  nativeBuildInputs = [
```
native should come before buildInputs.